mne/io/base.py

```diff
@@ -1180,13 +1181,17 @@ def filter(self, l_freq, h_freq, picks=None, filter_length='auto',
             then a minimum-phase, causal filter will be used.
 
             .. versionadded:: 0.13
-
         fir_window : str
             The window to use in FIR design, can be "hamming" (default),
             "hann" (default in 0.13), or "blackman".
 
             .. versionadded:: 0.13
+        fir_design : str
+            Can be "firwin" (default in 0.16) to use
+            :func:`scipy.signal.firwin`, or "firwin2" (default in 0.15 and
+            before) to use :func:`scipy.signal.firwin2`.
```

Comment:
Could this have the same description as the whats_new has? The difference isn't very clear.